src/Common/precompiled.h

```diff
@@ -210,11 +220,26 @@ typedef union _LARGE_INTEGER {
 
 // macros
 #if BOOST_OS_WINDOWS
-#define DLLEXPORT __declspec(dllexport)
+    #define DLLEXPORT __declspec(dllexport)
+    #define DLLIMPORT  __declspec(dllimport)
+    #define DEBUG_BREAK __debugbreak()
+    #define ALIGN(N) __declspec(align(N))
+    #define NOINLINE __declspec(noinline)
+    #define ASSUME(X) __assume((X)
+    #define THREAD_LOCAL __declspec(thread)
+    #define POPCNT(X) __popcnt((X))
 #else
-#define DLLEXPORT __declspec(dllexport)
+    #define DLLEXPORT
+    #define DLLIMPORT
+    #define DEBUG_BREAK
```

Comment:
I've never used break points in code (my code editor allows me to insert them at a specific line without altering the source), so the best I can do is link this [SO page](https://stackoverflow.com/questions/173618/is-there-a-portable-equivalent-to-debugbreak-debugbreak)

Reply:
I have added it in a separate PR

Reply:
I've realised `__debugbreak` was only used in `precompiled.h` so now I'm thinking it may be tidier to conditionally replace the cemu_assert_... functions.